docusaurus/docs/actors/relay_miner.md

```diff
@@ -0,0 +1,32 @@
+---
+title: RelayMiner
+sidebar_position: 5
+---
+
+# RelayMiner <!-- omit in toc -->
+
+- [Overview](#overview)
+- [Configuration](#configuration)
+- [CLI](#cli)
+
+## Overview
+
+An `RelayMiner` is responsible for for relaying requests between a client dApp
+(e.g. mobile app, web app, etc...) and the `Supplier`s on Pocket Network, handling
+all intermediary business logic.
+
+A `Gateway` operator or a sovereign `Application` interested in accessing Pocket
```

Comment:
This paragraph should talk about the `Supplier` interested in proposing `Services`